types/utils.d.ts

```diff
@@ -20,6 +20,11 @@ declare global {
         : { [K in keyof T]?: DeepPartial<T[K]> };
 
     type NoStringIndex<T> = { [K in keyof T as string extends K ? never : K]: T[K] };
+
+    type Prettify<T> = {
```

Comment:
This utility type is helpful for debugging your complex and nested types, It helps to see what's actually inside the type instead of showing nested type names. 

<img width="1024" alt="Screenshot 2023-05-07 at 1 45 35 PM" src="https://github.com/binary-com/deriv-app/assets/72082844/a6177996-4464-4147-a574-93687bfd47d3">